solr/core/src/java/org/apache/solr/cloud/api/collections/Assign.java

```diff
@@ -562,8 +566,33 @@ public static AssignStrategy createAssignStrategy(CoreContainer coreContainer) {
         coreContainer.getPlacementPluginFactory().createPluginInstance();
     if (placementPlugin == null) {
       // Otherwise use the default
-      // TODO: Replace this with a better options, such as the AffinityPlacementFactory
-      placementPlugin = (new SimplePlacementFactory()).createPluginInstance();
+      String defaultPluginId = System.getProperty(DEFAULT_PLACEMENT_PLUGIN_SYSPROP);
+      if (defaultPluginId != null) {
+        switch (defaultPluginId) {
```

Comment:
Can you do a `.toLowercase()` here? Thinking about ease-of-use for the `minimizeCores` option.

Reply:
Done